### DISCUSSION TITLE
Podman containers & virtual machines

### DISCUSSION THREAD

**Discussion by guercheLE** [cockpit-project/cockpit#18331] [Q&A]
Hi you all out there,

I am running Podman containers and virtual machines cockpit plugins. If I open console on any container and ping a virtual machine's ip, it is reachable. If I ping a container ip from a virtual machine it is unreachable. Anyone to the rescue on how to let a container reach an vm ip? What I am trying to achieve is let a Guacamole container reach  xrdp or vnc running on a linux VM.

**Comment by martinpitt** [comment]
Do you run system or user containers? User containers only support userspace emulated networking with "slirp", you can't reach them from outside. Inside they have address "10.0.2.2". For ingress routing you need a proper bridge on the host, with the "bridge" driver; system containers do that by default, they get an address like "10.88.0.2".

**Reply by guercheLE** [reply]
I run them as system containers. I run both containers and VMs on bridge networks. Container's IP 10.89.0.33 and VM's IP 192.168.122.151
